LayoutTests/imported/w3c/web-platform-tests/html/links/301-response.py

```diff
@@ -0,0 +1,27 @@
+def main(request, response):
+    response.status = 200
+    headers = [(b"Content-Type", b"text/html")]
+    return headers, u'''
+<!DOCTYPE html>
+<html>
+<title>Ensure fragments are correctly transferred on redirects</title>
+<body>
+    <script src='/resources/testharness.js'></script>
+    <script src='/resources/testharnessreport.js'></script>
+    <script>
+        if (window.location.href.includes("#key=foo")) {
+            element = document.createElement('p');
+            element.innerText = "SUCCESS";
+            document.body.appendChild(element);
+        } else {
+            element = document.createElement('p');
+            element.innerText = "FAILURE";
+            document.body.appendChild(element);
+        }
+
+        if (window.testRunner)
+            testRunner.notifyDone();
```

Comment:
We cannot have testRunner in WPT tests, this might not be needed anyway given we are using testharness.js.
Please move them to http/wpt for now, or please upstream these changes to WPT.

Reply:
Thanks. Removed.